packages/nuxt-module/src/module.ts

```diff
@@ -85,6 +85,11 @@ export default defineNuxtModule<ModuleOptions>({
 
     logger.verbose('🔌  Storybook Module Setup')
 
+    if (nuxt.options.dev) {
+      // Enable runtimeCompiler for Vue to support templates in stories
+      nuxt.options.vue.runtimeCompiler = true
+    }
```

Comment:
is it working cause when i tested the custom template stories on showcase example, was not rendering, actually in my last PR https://github.com/nuxt-modules/storybook/pull/830 , i added resolver to Vite configuration to  alias "vue" to "vue/dist/vue.esm-bundler.js

Reply:
i don't know if what i did still necessary after this PR